gnomad_qc/v4/create_release/create_release_sites_ht_exomes.py

```diff
@@ -492,8 +502,8 @@ def main(args):
         version=args.version,
     )
 
-    # The dbsnp table does not have a global field for dbsnp_versions, same
-    # with sift/polyphen and vrs (still need these)
+    # The dbsnp table does not have a global field for dbsnp_versions, same with
+    # sift/polyphen and vrs (still need these).
```

Comment:
```suggestion
            sift_version="5.2.2",
            polyphen_version="2.2.2",
            vrs_version=hl.struct(
                **{
                    "vrs_schema_version": "1.3.0"
                    "vrs_python_version": "0.8.4",
                    "seqrepo_version": "2018-11-26",
                },
            ),
        )
    )
```
It's for the lines below. 
How can I keep the indent without using suggestion?